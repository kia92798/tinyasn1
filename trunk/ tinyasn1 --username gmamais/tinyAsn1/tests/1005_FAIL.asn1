MY-MODULE DEFINITIONS AUTOMATIC TAGS::= BEGIN
	
	MyInt ::= INTEGER
	myIntVal INTEGER ::= 4294967296429496729642949672964294967296 -- TOO LARGE
	
	
END
