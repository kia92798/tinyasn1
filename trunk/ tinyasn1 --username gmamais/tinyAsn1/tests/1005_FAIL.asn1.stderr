Error in line : 4. Integer value is too large
